### DISCUSSION TITLE
Should NODE_PATH be modified before tests are run?

### DISCUSSION THREAD

**Discussion by hildjj** [avajs/ava#2682] [General]
ava 3.15 changed functionality that was introduced in #531 to ensure that the `NODE_PATH` environment variable would be kept intact for tests.

The difference can be seen running this test under ava@3.14 and ava@3.15:

```js
test('NODE_PATH', t => {
  t.regex(process.env.NODE_PATH, /foo/)
})
```

running with:

```sh
NODE_PATH=foo ava
```

Let's discuss if this is correct, and if it is, if any changes should be made to the documentation.

**Comment by hildjj** [comment]
And of course I can't reproduce this anymore; I think this might have been pnpm's fault.  Please feel free to close this discussion, and accept my apologies for the noise.

**Reply by novemberborn** [reply]
😄 

All references to `NODE_PATH` were removed in 3.0, not recently.
